content/en/docs/ops/configuration/traffic-management/network-topologies/index.md

```diff
@@ -0,0 +1,186 @@
+---
+title: Configuring Gateway Network Topology (Experimental)
+description: How to configure gateway network topology (experimental).
+weight: 60
+keywords: [traffic-management,ingress,gateway]
+---
+
+## (Experimental) Configuring network topologies
+
+Istio provides the ability to manage settings like X-Forwarded-For (XFF) and X-Forwarded-Client-Cert (XFCC), which are
+dependent on how the gateway workloads are deployed. This is currently an experimental feature.
+
+Many users choose to deploy Istio ingress gateways in using various network topologies
+(e.g. behind Cloud Load Balancers, a self-managed Load Balancer or directly expose the
+Istio ingress gateway to the internet). As such, these topologies require different ingress gateway configurations for
+transporting correct client attributes like IP/certs to the workloads running in the cluster.
+
+Configuration of XFF and XFCC headers can be configured by using `MeshConfig` during Istio
+*installation* or by adding a pod annotation.
+
+To simplify configuring network topology during installation create a single YAML file to pass to `istioctl`:
+
+{{< text yaml >}}
+cat <<'EOF' > topology.yaml
+apiVersion: install.istio.io/v1alpha1
+kind: IstioOperator
+spec:
+  meshConfig:
+    defaultConfig:
+      gatewayTopology:
+EOF
+{{< /text >}}
+
+{{< warning >}}
```

Comment:
change to idea. This would diminish the strength of the callout for warning since its really a suggestion. A warning is something like "your data will be permanently lost if you proceed." 👍

Reply:
Fixed.